api/config/esgateway/config_request.go

```diff
@@ -48,8 +48,9 @@ func DefaultConfigRequest() *ConfigRequest {
 	// resolvers once we are a bit more confident in the nginx
 	// module we are using to do dynamic resolution.
 	//
-	// c.V1.Sys.Ngx.Main.Resolvers = getSystemResolvers()
 
+	c.V1.Sys.Ngx.Main.Resolvers.EnableSystemNameservers = w.Bool(false)
+	c.V1.Sys.Ngx.Main.Resolvers.Nameservers = nil
```

Comment:
We can probably avoid setting this to nil since it will be nil by default when the struct is initialized

Reply:
Removed the code to set Nameservers as nil in the latest changes